### PR TITLE
V1.2 list

### PR DESCRIPTION
**Pull request by RoeReRe**
Added list command. Did it slightly different from AB3 in that I mandated `list` strictly.

Example: `list asdasdasd` works for AB3 but throws an error in BayMeds.

Also added **temporary** labels to the display so that its easier for us to see. I'll make them nicer (maybe color block it) in the future.

Understandably this addition amounts to naught. The repo is very messy atm so in the near future I'll work on the test codes and GUI if time permits so that we can remove the AB3 clutter and pass code cov.

### CODE REVIEW

**Review by shyanyong — APPROVED** [review]
lgtm

**Review by shyanyong — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/ListPrescriptionCommand.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import seedu.address.model.ModelPrescription;
+
+/**
+ * Lists all prescriptions stored to the user.
+ */
+public class ListPrescriptionCommand extends CommandPrescription {
+
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all prescriptions stored.\n"
```
Comment (ChongWeiJie29) [inline src/main/java/seedu/address/logic/commands/ListPrescriptionCommand.java:15]: Commenting it here for now:

We will need to standardise cuz currently some no fullstop, some have. 
Lets standardise as every sentence in the result text box will have a fullstop. So do change/add to it as ure doing ur features :>>

**Review by ChongWeiJie29 — APPROVED** [review]
lgtm

**Pull request merged**